src/common/components/Accordion/index.tsx

```
@@ -0,0 +1,121 @@
+  const { isOpen, toggle, accordionId } = useAccordionContext();
+  return (
+    <S.AccordionHeader
+      aria-expanded={isOpen}
+      aria-controls={accordionId}
+      aria-disabled={false}
+      onClick={toggle}
+      {...props}
+    >
+      <S.HeadingText as="h3">
+        {children}
+      </S.HeadingText>
```

ESLint

[error] Replace `⏎········{children}⏎······` with `{children}`

(prettier/prettier)